### PR TITLE
Feature/jags

### PR DESCRIPTION
**Pull request by zenln**

### CODE REVIEW

File: var/spack/repos/builtin/packages/jags/package.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    url = "https://downloads.sourceforge.net/project/mcmc-jags/JAGS/4.x/Source/JAGS-4.2.0.tar.gz"
+
+    version('4.2.0', '9e521b3cfb23d3290a8c6bc0b79bf426')
+
+    depends_on('m4', type='build')
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
+
+    depends_on('atlas')
```
Comment (tgamblin) [inline var/spack/repos/builtin/packages/jags/package.py]: Does it depend specifically on atlas or can it run with any `blas` implementation?  If the latter, can you make it depend on `blas`?

**Comment by zenln** [comment]
Changed depends on to just include netlib-lapack.

File: var/spack/repos/builtin/packages/jags/package.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    url = "https://downloads.sourceforge.net/project/mcmc-jags/JAGS/4.x/Source/JAGS-4.2.0.tar.gz"
+
+    version('4.2.0', '9e521b3cfb23d3290a8c6bc0b79bf426')
+
+    depends_on('m4', type='build')
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
+
+    depends_on('netlib-lapack')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/jags/package.py]: Does it only work with `netlib-lapack` or can it work with any LAPACK? If the latter, make it `depends_on('lapack')`.

Comment (zenln) [inline var/spack/repos/builtin/packages/jags/package.py]: Did not work with lapack.

Comment (adamjstewart) [inline var/spack/repos/builtin/packages/jags/package.py]: What do you mean it did not work? `blas` and `lapack` are virtual packages, of which there are multiple different packages such as `atlas` and `netlib-lapack` that provide them. Perhaps some work and some don't?

Comment (zenln) [inline var/spack/repos/builtin/packages/jags/package.py]: jags need lapack we netlib-lapack/atlas both work.

Comment (adamjstewart) [inline var/spack/repos/builtin/packages/jags/package.py]: So you're saying that any LAPACK provider should work, but in practice only `netlib-lapack` and `atlas` seem to work? In that case, I would use:
```python
depends_on('lapack')

# Only netlib-lapack and atlas work
conflicts('^intel-mkl')
conflicts('^intel-parallel-studio')
conflicts('^openblas')
conflicts('^veclibfort')
```

**Comment by adamjstewart** [comment]
So... does it depend on atlas or netlib-lapack?

**Comment by zenln** [comment]
Depends on atlas.

**Pull request closed without merging**